modules/schema/pages/notices-xml-structure.adoc

```diff
@@ -87,3 +87,23 @@ the procurement Procedure;
 
 * as children of a Procurement Project Lot element, they refer to the
 corresponding Group of Lots, Lot or Part.
+
+== Business Terms
+
+The Business Terms listed in Annex II of the eForms regulation are described
+in this document, and use the same identifiers beginning with "BT-".
+
+Basing the development of the eForms schema on the UBL schema, as well as 
+conferring many advantages, has also imposed some constraints. These 
+constraints have required the creation of a number of terms which were not 
+anticipated in the eForms regulations; they do not have a true Business 
+justification. They have been assigned different abbreviations to distinguish 
+them from the BT terms defined in the eForms regulations.
+
+Two abbreviations for these fields have been introduced: "OPP" and "OPT". "OP" 
+is the abbreviation for the Publications Office of the European Union as 
+defined above. "P" stands for Production; these fields are required for the 
+production processes, particularly for the non-standard forms (not defined in 
+the eForms regulations) that also use the eForms schema. "T" stands for 
+Technical, these are required by our use of UBL as the base schema for eForms.
```

Comment:
Instead of "by our use of UBL as the base schema for eForms.", I would suggest "following some technical constraints and/or chosen design options." or anything equivalent.

Reply:
Thanks Yves. I've updated that sentence.